oak/proto/enclave.proto

```diff
@@ -26,6 +26,9 @@ message InitializeInput {
   // See https://webassembly.org/docs/binary-encoding/.
   optional bytes module = 1;
   optional string node_id = 2;
+  // The port number that the enclave's GRPC server listens on; set this to 0
```

Comment:
I think we should support only one way to assign ports; in particular, now that we have this logic, I think we should explicitly disallow setting this to 0?

Reply:
Sounds good, 0 is now disallowed